exporter/src/main/as/flump/export/MaxRectPackerImpl.as

```diff
@@ -0,0 +1,166 @@
+package flump.export {
```

Comment:
This is a copypaste from https://github.com/treefortress/TextureUtils/blob/master/src/treefortress/textureutils/MaxRectPacker.as